### ISSUE TITLE
Dynamic shortcuts in search results

### ISSUE THREAD

**Issue by Wendigogo** [TBog/TBLauncher#259]
**Description**

**Solution**

I searched on settings (and in GitHub issues) but didn't find if this is already available or demanded somewhere.

Enjoy!

**Comment by TBog** [comment]
Shortcuts already apper in the Search results. What is missing?
Dynamic shortcuts can be seen by long-touching the app in the Result list.

**Comment by Wendigogo** [comment]
That's it: Dynamic shortcuts.
Could they be listed in search results directly, like in KISS launcher please?

**Comment by Wendigogo** [comment]
Sorry for the long silence. 

Here are screenshots to illustrate what I talked about.

![Screenshot_20220307-100005_KISS launcher](https://user-images.githubusercontent.com/1914251/157000218-be7fdf92-a05a-4ab7-b5d8-a8e705fa32e4.jpg)
With Kiss Launcher I have directly Dynamic shortcuts.

Here is what I got with Tinybit Launcher:
![Screenshot_20220307-100023_TinyBit launcher](https://user-images.githubusercontent.com/1914251/157000459-bb2b5586-28f8-4873-8323-356f552f6676.jpg)
I could reach these shortcuts with longpress but I would prefer have them on search results as it's faster and easier.

I think longpress should stay as it's a good way to access all dynamic shortcuts (some apps have many) but an option to have them appearing on fuzzy search would be great to have those oftenly used.

Thanks for your support.
